### ISSUE TITLE
Bevel Tool: Choose radius to follow mouse pointer

### ISSUE THREAD

**Issue by hlorus** [hlorus/CAD_Sketcher#275]
Currently when adding a bevel the radius will be the distance between the mouse pointer and the corner that is being beveled. This can be quite annoying in some cases e.g. when creating a bevel between two lines with a small angle in between as the radius can easily get too big and lead to a failure of the operation.
A nice improvement would be to compute the radius so that the created arc goes through the mouse pointer.

**Comment by amrsoll** [comment]
I don't believe it is achievable with current method, but trickery can be achieved! :smiling_imp: 

For a circular bevel: this requires us to map a point on one of the lines to the centerpoint, or we would need to calculate a path along which the centerpoint 

For a flat bevel (not implemented):  that should be manageable because we don't need to recalculate the orientation of it (I think?)

For a NURB-style bezel (not implemented): Same as the flat bevel, maybe even easier as we don't even have to calculate angles, just need 3 positions.

**Trickery**:

We can keep the current method, but simply multiply the radius by the sine of connecting angle. Far from perfect for acute angles with arcs though.

**Comment by hlorus** [comment]
The bevel tool's radius is now interactive — it follows the cursor while placing the bevel. Closing as implemented.
